autode/utils.py

```diff
@@ -36,12 +41,12 @@ def wrapped_function(*args, **kwargs):
         physical_mem = None
         required_mem = int(Config.n_cores) * Config.max_core
 
-        try:
+        try:  # check available memory
```

Comment:
```suggestion
        try:
```
inline comments are bad, in my opinion at least! If it's not obvious what the code is doing then the control flow or naming should be changed appropriately – comments become wrong over time 😢